app/src/main/java/com/trainstation/frettirtest2/MainActivity.java

```diff
@@ -14,23 +14,22 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    ViewPager pager;
-    ViewPageAdapter adapter;
+    private ViewPager pager;
+    private ViewPageAdapter adapter;
     private DrawerLayout mDrawerLayout;
     private ListView mDrawerList;
     private String[] mPlanetTitles;
 
-    NewsService newsService;
+    private NewsService newsService;
```

Comment:
it is always good practice to declare the visibility of the variables, and let them be as invisible as you can